samples/server/petstore/jaxrs-resteasy/eap-java8/pom.xml

```diff
@@ -34,23 +46,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>exec-maven-plugin</artifactId>
-                <version>1.6.0</version>
-                <executions>
-                    <execution>
-                        <id>Play Test</id>
-                        <phase>integration-test</phase>
-                        <goals>
-                            <goal>exec</goal>
-                        </goals>
-                        <configuration>
-                            <executable>./mvn_test_jdk8_only.sh</executable>
```

Comment:
Right. I actually can think of a better way to avoid using shell script to run only with JDK8. I'll do it later after this PR.